CONTRIBUTING.rst

```diff
@@ -3,8 +3,11 @@ Contributing to MLflow
 We welcome community contributions to MLflow. This page describes how to develop/test your changes
 to MLflow locally.
 
+Python API
```

Comment:
Maybe we could say what components exist in which languages, to help people understand? For example, we could change to this to just say "Python" and have a sentence like, "The majority of the MLflow codebase is in Python. This includes the CLI, Tracking Server, Artifact Repositories (e.g., S3 or Azure Blob Storage backends), and of course the Python fluent, tracking, and model APIs. It would be even better if we could link to docs and/or code for each of these, but just the description by itself would be great.